docs/usage-examples/updateOne.txt

```diff
@@ -0,0 +1,109 @@
+.. _laravel-update-one-usage:
+
+=================
+Update a Document
+=================
+
+You can update a document in a collection by chaining the ``where()``, ``first()``,
+and ``update()`` methods to an Eloquent model or a query builder.
+
+Pass a query filter to the ``where()`` method and call the ``first()`` method to match 
+only one document. Then, pass the document changes to the ``update()`` method to update
+one document that matches the filter.
+
+To learn more about updating data with {+odm-short+}, see the `Updates
+<https://laravel.com/docs/6.x/eloquent#updates>`__ section of the Laravel documentation.
```

Comment:
You can link to the latest version (currently 11.x) by removing the version from the URL.
```suggestion
<https://laravel.com/docs/eloquent#updates>`__ section of the Laravel documentation.
```